frontend/src/main/components/BasicCourseSearch/BasicCourseSearchForm.js

```diff
@@ -37,10 +38,29 @@ const BasicCourseSearchForm = ({ fetchJSON }) => {
   const [quarter, setQuarter] = useState(localQuarter || quarters[0].yyyyq);
   const [subject, setSubject] = useState(localSubject || {});
   const [level, setLevel] = useState(localLevel || "U");
+  const [courseNumber, setCourseNumber] = useState(localCourseNumber || "");
+  const [courseSuf, setCourseSuf] = useState("");
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    fetchJSON(event, { quarter, subject, level });
+    fetchJSON(event, { quarter, subject, level, courseNumber, courseSuf });
+  };
+
+  const handleCourseNumberOnChange = (event) => {
+    const rawCourse = event.target.value;
+    if (rawCourse.match(/\d+/g) != null) {
+        const number = rawCourse.match(/\d+/g)[0];
+        setCourseNumber(number);
+    } else {
+        setCourseNumber("");
+    }
+    
+    if (rawCourse.match(/[a-zA-Z]+/g) != null) {
+        const suffix = rawCourse.match(/[a-zA-Z]+/g)[0];
+        setCourseSuf(suffix);
+    } else {
+        setCourseSuf("");
+    }
   };
 
   // Stryker disable all : Stryker is testing by changing the padding to 0. But this is simply a visual optimization as it makes it look better
```

Comment:
The stryker disable all here is not ok... we can write a test for the CSS, and even if we couldn't disabling all stryker mutation testing for the entire component leaves us vulnerable to lots of testing gaps.

Please add a test for the CSS property, and remove this.

I'm aware that it was in the component before, but we need to address this tech debt.

Reply:
Especially since you are changing code in the area that this test exception covers.

Reply:
I'm a bit confused on how to go about making a test for a CSS property. I believe this stryker disable was in the starter code. Is there an example of a test somewhere else in the starter code that I can base this off of?

Reply:
Here's a link on how to do it: 

https://ucsb-cs156.github.io/topics/stryker/stryker_html_attribute_surviving_mutants.html